config/locales/js-en.yml

```diff
@@ -575,6 +575,8 @@ en:
         unread: 'Unread'
         all: 'All'
       center:
+        and_other_users_singular: 'and 1 other'
+        and_other_users_plural: 'and %{count} others'
```

Comment:
Prefer to use a nested key as that will create a grouped translation key in crowdin and easier to understand for pluralization. Something like https://github.com/opf/openproject/blob/dev/config/locales/js-en.yml#L1102-L1105